### DISCUSSION TITLE
Installation on Proxmox

### DISCUSSION THREAD

**Discussion by pgharios** [Security-Onion-Solutions/securityonion#12326] [2.4]
### Version

2.4 Pre-release (Beta, Release Candidate)

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

cloud

### Hardware Specs

Meets minimum requirements

### CPU

2

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

tap

### Network Traffic Speeds

more than 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Hello,

Anyone got SO working with Proxmox in Desktop mode with GUI?

I am getting a blank screen when doing so and booting into Desktop/GUI.

Appreciate to share proxmox configuration if anyone got it working successfully.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by robbiemarshall** [comment]
Have you followed all of the recommendations in the documentation here:  https://docs.securityonion.net/en/2.4/proxmox.html
